src/storage/src/storage/signed_url.rs

```diff
@@ -126,6 +128,15 @@ impl SigningScope {
     }
 }
 
+// Used to check conformance test expectations.
+struct SigningComponents {
+    #[allow(dead_code)]
```

Comment:
Is it possible to use cfg(test) for these instead too? If that's what these are used for.